src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/VariantRecalibrator.java

```diff
@@ -0,0 +1,852 @@
+     * The expected transition / transversion ratio of true novel variants in your targeted region (whole genome, exome, specific
+     * genes), which varies greatly by the CpG and GC content of the region. See expected Ti/Tv ratios section of the GATK best
+     * practices documentation (http://www.broadinstitute.org/gatk/guide/best-practices) for more information.
+     * Normal values are 2.15 for human whole genome values and 3.2 for human whole exomes. Note
+     * that this parameter is used for display purposes only and isn't used anywhere in the algorithm!
+     */
+    @Argument(fullName="target_titv",
+            shortName="titv",
+            doc="The expected novel Ti/Tv ratio to use when calculating FDR tranches and for display on the optimization curve output figures. (approx 2.15 for whole genome experiments). ONLY USED FOR PLOTTING PURPOSES!",
+            optional=false)
+    private double TARGET_TITV = 2.15;
+
+    /**
+     * See the input VCF file's INFO field for a list of all available annotations.
+     */
+    @Argument(fullName="use_annotation",
+            shortName="an",
+            doc="The names of the annotations which should used for calculations",
+            optional=false)
+    private List<String> USE_ANNOTATIONS = new ArrayList<>();
+
+    /**
+     * Add truth sensitivity slices through the call set at the given values. The default values are 100.0, 99.9, 99.0, and 90.0
+     * which will result in 4 estimated tranches in the final call set: the full set of calls (100% sensitivity at the accessible
+     * sites in the truth set), a 99.9% truth sensitivity tranche, along with progressively smaller tranches at 99% and 90%.
+     */
+    @Argument(fullName="TStranche",
+            shortName="tranche",
+            doc="The levels of truth sensitivity at which to slice the data. (in percent, that is 1.0 for 1 percent)",
+            optional=true)
+    private List<Double> TS_TRANCHES = new ArrayList<Double>(Arrays.asList(100.0, 99.9, 99.0, 90.0));
+
+    /**
+     * For this to work properly, the -ignoreFilter argument should also be applied to the ApplyRecalibration command.
+     */
+    @Argument(fullName="ignore_filter",
+            shortName="ignoreFilter",
+            doc="If specified, the variant recalibrator will also use variants marked as filtered by the specified filter name in the input VCF file",
+            optional=true)
+    private List<String> IGNORE_INPUT_FILTERS = new ArrayList<>();
+
+    @Argument(fullName="ignore_all_filters",
+            shortName="ignoreAllFilters",
+            doc="If specified, the variant recalibrator will ignore all input filters. Useful to rerun the VQSR from a filtered output file.",
+            optional=true)
+    private boolean IGNORE_ALL_FILTERS = false;
+
+    @Argument(fullName="rscriptFile", shortName="rscriptFile", doc="The output rscript file generated by the VQSR to aid in visualization of the input data and learned model", optional=true)
```

Comment:
Preserve the old `fullName` (`rscript_file`) unless there's a good reason not to.